sam-test-project/tests/snapshot/sam-test-project-snapshot.test.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from 'tap'
+import type { Template } from 'cloudform'
+import { handler } from 'cf-macro-slic-watch/index'
+
+import inputTemplate from './fixtures/sam-test-project-transformed-template.json'
+import { setUpSnapshotDefaults } from 'test-utils/snapshot-utils'
+
+test('sam-test-project snapshot', async (t) => {
```

Comment:
I usually use tests as the first point of documentation, having test names that explains the test name helps confirm the initial state and assertion. It's not obvious to me what this test is testing, I need to read the test function and and the application code to understand. BDD type tests are not necessary but the format could be something like 
```typescript
test('a good sam template generates successful slic watch dashboard and alarms') 
``` 

Its no a big issue though

Reply:
Good advice, taking it!